src/os/darwin/darwin_sigar.c

```diff
@@ -29,6 +29,7 @@
 #include <nfs/rpcv2.h>
 #include <nfs/nfsproto.h>
 #include <nfs/nfs.h>
+#ifndef NFS_NPROCS
 #if (__MAC_OS_X_VERSION_MIN_REQUIRED >= 101406)
```

Comment:
```suggestion
#if (__MAC_OS_X_VERSION_MAX_ALLOWED >= 120000)
```

Found out about this another constant. Reverting to original guard. This will only run if on MacOSX12.sdk or above which serves our purpose. Tested this change with 10.15.6, 11.3, 12.3, & 13.1 SDK. Compiles normally. I think we don't need the macOS version guard because this issue only depends on the header definition of nfsstats in the SDK so only need to check for that.

Reply:
Thank you that is looks like a great solution.